app/pages/organization/stats/components/ProjectStats/components/Workflows/WorkflowsShowing.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Container from './components/Container';
+import ToggleButton from './components/ToggleButton';
+import Section from './components/Section';
+import Heading from '../Heading';
+import ProgressBar from '../ProgressBar';
+
+function WorkflowsShowing({ project, toggleWorkflows }) {
+  return (
+    <Container>
+      <ToggleButton
+        icon={<i className="fa fa-chevron-up fa-lg" />}
+        handleClick={toggleWorkflows}
+      />
+      <Section>
+        {project.workflows.map(workflow => (
+          <div key={workflow.id}>
```

Comment:
Same as above. Can this key be `WORKFLOW_${workflow.id}` or something similar to avoid the off chance of conflicting ids?